dev-docs/source/Testing/SANSGUI/ISISSANSGUITests.rst

```diff
@@ -124,8 +124,10 @@ Processing
 #. In the workspaces list, there should be a series of new workspaces; four
    group workspaces and four 1D workspaces.
 #. Check your default save directory. For each reduction two banks (HAB/main) should
-   be saved. In total there should be 12 workspaces (6 CanSAS ``.xml`` and 6 NXcanSAS ``.h5``) saved. For each row and
-   file type there should be a reduced file (with no suffix), a sample, and a can workspace.
+   be saved. In total there should be 20 workspaces saved. For each row, file type, and bank there should be a
```

Comment:
be saved. In total, there should be 20 workspaces saved. For each row, file type, and bank, there should be a

Reply:
I’m aware that I’m nit-picking, and I apologize for it.